File: VB/MessengerExample/Model/EmployeeContextInitializer.vb
Imports System.Collections.Generic
Imports System.Data.Entity

Namespace MessengerExample.Model
    Public Class EmployeeContextInitializer
        Inherits DropCreateDatabaseIfModelChanges(Of EmployeeContext)

        Protected Overrides Sub Seed(ByVal context As EmployeeContext)
            MyBase.Seed(context)
            Dim employees As New List(Of Employee)() From {New Employee("John", "Smith")}
            employees.ForEach(Sub(x) context.Employees.Add(x))
            context.SaveChanges()
        End Sub
    End Class
End Namespace

File: CS/MessengerExample/Model/EmployeeContextInitializer.cs
using System.Collections.Generic;
using System.Data.Entity;

namespace MessengerExample.Model {
    public class EmployeeContextInitializer : DropCreateDatabaseIfModelChanges<EmployeeContext> {
        protected override void Seed(EmployeeContext context) {
            base.Seed(context);
            List<Employee> employees = new List<Employee>() {
                new Employee("John", "Smith"),
            };
            employees.ForEach(x => context.Employees.Add(x));
            context.SaveChanges();
        }
    }
}
